marimo/_server/print.py

```diff
@@ -64,10 +64,30 @@ def _get_network_url(url: str) -> str:
 
     hostname = socket.gethostname()
     try:
-        local_ip = socket.gethostbyname(hostname)
+        # Find a non-loopback IPv4 address
+        s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+        # Doesn't need to be reachable
+        s.connect(("8.8.8.8", 1))
```

Comment:
8.8.8.8 is a google server, is 255.255.255.254 OK? 240-255 is unassigned IPV4 space